extension-config.json

```
@@ -0,0 +1,60 @@
+{
+  "version": "1.0.0",
+  "id": "@shopgate-project/ext-pdp-accordion",
```

only the repo name should have the "ext-" prefix
```suggestion
  "id": "@shopgate-project/pdp-accordion",
```

---

done